src/main/scala/org/hammerlab/guacamole/loci/partitioning/LociPartitioner.scala

```diff
@@ -0,0 +1,58 @@
+package org.hammerlab.guacamole.loci.partitioning
+
+import org.apache.spark.rdd.RDD
+import org.hammerlab.guacamole.loci.LociArgs
+import org.hammerlab.guacamole.loci.set.LociSet
+import org.hammerlab.guacamole.reference.ReferenceRegion
+import org.hammerlab.magic.args4j.StringOptionHandler
+import org.kohsuke.args4j.{Option => Args4JOption}
+
+import scala.reflect.ClassTag
+
+trait LociPartitionerArgs
+  extends LociArgs
+  with ApproximatePartitionerArgs
+  with UniformPartitionerArgs {
+
+  @Args4JOption(
+    name = "--loci-partitioning",
+    usage = "Directory path within which to save the partitioned reads and accompanying LociMap partitioning.",
+    forbids = Array("--partitioning-dir"),
```

Comment:
Is `partitioning-dir` used?

Also might be helpful to clarify if this is only an output parameter (i.e., it's only written but never read, in which case maybe call it something like ``--out-loci-partitioning``) or both an output-and-an-input argument. Also from the name alone it would seem to me it's saving only loci, so if it's also saving reads another name (`--out-partitioned-reads`?) might be appropriate